test/Hyperwallet.spec.js

```diff
@@ -4904,4 +4904,90 @@ describe("Hyperwallet", () => {
             });
         });
     });
+
+    /** @test {Hyperwallet#getTransferStatusTransition} */
+    describe("getTransferStatusTransition()", () => {
+        let client;
+        let apiClientSpy;
+        beforeEach(() => {
+            apiClientSpy = sinon.spy();
+            client = new Hyperwallet({
+                username: "test-username",
+                password: "test-password",
+            });
+            client.client = {
+                doGet: apiClientSpy,
+            };
+        });
+        /** @test {Hyperwallet#getTransferStatusTransition} */
+        it("should throw error if transferToken is missing", () => {
+            const callback = () => null;
+            expect(() => client.getTransferStatusTransition(undefined, undefined, callback)).to.throw("transferToken is required");
+        });
+        /** @test {Hyperwallet#getTransferStatusTransition} */
+        it("should throw error if statusTransitionToken is missing", () => {
+            const callback = () => null;
+            expect(() => client.getTransferStatusTransition("test-transfer-token", undefined, callback)).to.throw("statusTransitionToken is required");
+        });
+        /** @test {Hyperwallet#getTransferStatusTransition} */
+        it("should do get call if transferToken and statusTransitionToken is provided", () => {
+            const callback = () => null;
+            client.getTransferStatusTransition("test-transfer-token", "status-transition-token", callback);
```

Comment:
change status-transition-token to test-status-transition-token to differentiate test data from other constants.